src/feeds/direct-inbox.feed.ts

```diff
@@ -15,25 +21,25 @@ export class DirectInboxFeed extends Feed<DirectInboxFeedResponse, DirectInboxFe
     this.cursor = body.inbox.oldest_cursor;
   }
 
-  async request() {
+  async request(opts?: DirectInboxFeedItemsOptions) {
```

Comment:
These options should live in the feed (the class). They may be supplied through `ig.feed.directInbox({here})`.

It should look something like this:

```ts
class MyFeed {
  option1: string;
  defaultOption = 'my value';

  request() {
    // use this.option1 and this.defaultOption
  }
}

const myFeed = ig.feed.myFeed({option1: 'a'});
```